src/encoded/static/components/experiment.js

```diff
@@ -661,6 +662,25 @@ const ExperimentComponent = ({ context, auditIndicators, auditDetail }, reactCon
                                 </div>
                             : null}
 
+                            {context.examined_loci && context.examined_loci.length > 0 ?
+                                <div data-test="examined-loci">
+                                    <dt>Examined Loci</dt>
+                                    <dd>
+                                        {/* A user can have a loci repeat. Therefore, uuid alone is not sufficient as an identifier */}
+                                        {context.examined_loci.map((loci, i) => (
+                                            loci.gene ?
+                                                <span key={[loci.gene.uuid, i].join('')}>
```

Comment:
A string template might look simpler, but probably makes no real difference.

Reply:
It was ES5 in me that creeped up. I updated this.